### ISSUE TITLE
GradientCumulativeOptimizerHook may slightly decrease performance if the model has BatchNorm layers.

### ISSUE THREAD

**Issue by simplify23** [open-mmlab/mmcv#1869]
i just have 10GB for each gpu, which set batch size 128. I think higher batch size can improve acc. I find GradientCumulativeOptimizerHook solve this problem, but i acquire bad acc (decrease 8%) and this warning `GradientCumulativeOptimizerHook may slightly decrease performance if the model has BatchNorm layers.`. Sure, i have BN in my model, can i change some code still use this api for improve my acc?

**Comment by mzr1996** [comment]
Here are some reference:
https://ai.stackexchange.com/questions/21972/what-is-the-relationship-between-gradient-accumulation-and-batch-size
&
https://discuss.pytorch.org/t/gradient-accumulation-with-batchnorm2d-layers/89245

**Comment by simplify23** [comment]
I apply transformer as my model, which has many BN layers. As I see in reference, can I change BN to instanceNorm or GroupNorm? i consider it will not obtain higher acc. Can't i use this GradientCumulativeOptimizerHook or if i set lower iteration will solve this issue?

**Comment by zhouzaida** [comment]
I think it doesn't have improvement to change BN to IN or GN.
